lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
The warning should only show if the assets are fingerprinted and there is not asset map, right? That would be a bad state. If they aren't fingerprinted, then they would be at their expected paths, and the asset map isn't required (As least that's my understanding)